backend/app/schemas/experiment_template.py

```diff
@@ -46,6 +46,7 @@ class ExperimentTemplateBase(BaseModel):
     available_metrics: list[str]
     script: str
     pip_requirements: str
+    is_public: bool
```

Comment:
I think it would be better to stick with simple standard terms `approved`, `public` and `archived`.